src/main/java/step3/RacingEvent.java

```diff
@@ -0,0 +1,31 @@
+package step3;
+
+import java.util.ArrayList;
+
+public class RacingEvent {
+
+    private ArrayList<Car> carList = new ArrayList<>();
```

Comment:
인스턴스 변수 이름에 명확한 구현체 이름을 담는 건 좋지 않습니다.
```suggestion
    private List<Car> cars = new ArrayList<>();
```
로 하는건 어떨까요?

Reply:
반영하였습니다.